x-pack/platform/plugins/shared/osquery/package.json

```diff
@@ -7,13 +7,13 @@
   "scripts": {
     "cypress:burn": "yarn cypress:run --env burn=2 --headed",
     "cypress:changed-specs-only": "yarn cypress:run --changed-specs-only --env burn=2",
-    "cypress": "NODE_OPTIONS=--openssl-legacy-provider node ../../../../solutions/security/plugins/security_solution/scripts/start_cypress_parallel --config-file ../../platform/plugins/shared/osquery/cypress/cypress.config.ts --ftr-config-file ../../../../../x-pack/test/osquery_cypress/cli_config",
+    "cypress": "NODE_OPTIONS=--openssl-legacy-provider node ../../../../solutions/security/plugins/security_solution/scripts/start_cypress_parallel --config-file ../../../x-pack/platform/plugins/shared/osquery/cypress/cypress.config.ts --ftr-config-file ../../../../../x-pack/test/osquery_cypress/cli_config",
```

Comment:
I am still confused about these path changes, but as long as it works, it's fine :) Thanks!

Reply:
I'm adding the `x-pack` bit so that if we relocate the module in the future... references to the path are easier to spot.